packages/components/src/Badge/Badge.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { createCssClassNames } from '@ibexa/ids-core/helpers/cssClassNames';
+
+import type { BadgeProps } from './Badge.types';
+
+const MAX_BADGE_VALUE = 99;
+
+const formatBadgeValue = (value: number): string => {
```

Comment:
you can put this method inside Badge body, just use useCallback to memoize it in order not to redeclararing it on each refresh